### DISCUSSION TITLE
how to stream and record offline together

### DISCUSSION THREAD

**Discussion by iamjpsharma** [HaishinKit/HaishinKit.swift#1052] [Q&A] [answered]
Hi, I need to record offline for that I'm using .localRecord and for live I'm using .live 
I need  to stream and record offline both 
how to implement this?

**Comment by shogo4405** [comment, marked as answer]
Currently version has no feature. Next version 1.3.0 has feature, but not released yet.
```
rtmpStream.startRecording()
rtmpStream.publish("name", .live)
```

**Reply by iamjpsharma** [reply]
Value of type 'RTMPStream' has no member 'startRecording'

**Reply by iamjpsharma** [reply]
Can’t find this method

**Reply by shogo4405** [reply]
Please try `main` branch.

**Reply by iamjpsharma** [reply]
Thanks 🙏
